src/renderer/containers/SendCoinsModal/index.tsx

```diff
@@ -96,6 +96,7 @@ const SendCoinsModal: FC<ComponentProps> = ({close, initialRecipient, initialSen
         .number()
         .callbackWithRef(senderAccountNumberRef, checkCoinsWithBalance, INVALID_AMOUNT_ERROR)
         .moreThan(0, 'Coins must be greater than 0')
+        .lessThan(9000000000000000, 'Coins must be lower than 9,000,000,000,000,000') // Less than Number.MAX_SAFE_INTEGER
```

Comment:
The same thing ( https://github.com/thenewboston-developers/Account-Manager/pull/457 )

Reply:
From my testing there is also an issue with the amount being over 999999999999999. as it will begin to round up. That being said I think a better approach may just be a limit from a managing stand point that the max number of coins a person can send would be like 100Mil or something.

Reply:
```suggestion
        .lessThan(100000000, 'Coins must be lower than 100,000,000')
```

Reply:
> 
> 
> From my testing there is also an issue with the amount being over 999999999999999. as it will begin to round up. That being said I think a better approach may just be a limit from a managing stand point that the max number of coins a person can send would be like 100Mil or something.

I agree with you on having a max limit at 100M. That's a huge amount and seems like a realistic limit per transaction. Multiple transactions would be possible anyway, if needed.

Reply:
> 
> 
> @giacomo-b
> I agree that the coin limit must be less than Number.MAX_SAFE_INTEGER (9,007,199,254,740,992). I don't think "coin limit" means just "variable limit". It might make another issue on the next step for us to use Number.MAX_SAFE_INTEGER. What do you think?
> Why can't we take a more accurate limit for coins? Did you have a chance to check this comment?
> [#438 (comment)](https://github.com/thenewboston-developers/Account-Manager/issues/438#issuecomment-727256170)

Sorry for the delay, I've been extremely busy. I agree, let's cap at 100M so that there are no problems down the pipeline. That sounds like a perfectly reasonable limit.